### DISCUSSION TITLE
PHP Error was encountered

### DISCUSSION THREAD

**Discussion by MH606** [magicbug/Cloudlog#1708] [General]
Just installed Cloudlog and imported my log from QRZ, on selecting Logbook from the menu I get the following 
A PHP Error was encountered
Severity: Notice

Message: Trying to access array offset on value of type bool

Filename: controllers/Logbook.php

Line Number: 62

Also another error as above but with Line Number 63 

Under this the logbook shows ok though

**Comment by AndreasK79** [comment]
That is strange, but looking at the logbook controller on line 62 and 63, it is referred to user locator:

	$data['qra_lat'] = $qra_position[0];
	$data['qra_lng'] = $qra_position[1];

Make sure your user locator is correctly set.

**Reply by G0POA** [reply]
Location is set I’m afraid

**Reply by AndreasK79** [reply]
Ok, then I really don't know. Did you set up a logbook and assign station locations to this logbook?

Which PHP version are you running?

**Comment by MH606** [comment]
Yes to both and PHP 7.4 but can upgrade if you think that’ll help.  I’ll do a clean install and create a new db it’s easy enough On 27 Oct 2022, at 11:35, Andreas Kristiansen ***@***.***> wrote:﻿
Ok, then I really don't know. Did you set up a logbook and assign station locations to this logbook?
Which PHP version are you running?

**Comment by magicbug** [comment]
Not sure it would, can you give us screenshots of the gridsquare field in your user account and also the active station location

**Comment by G0POA** [comment]
sure
________________________________

**Comment by magicbug** [comment]
Another thing that may be worth checking is the grid square in /application/config/config.php and make sure that isn't corrupt although that shouldn't cause such errors.

**Reply by phl0** [reply]
Just removed grid square on my test system from application/config/config.php as well as from the active station location. Does not result in the error mentioned ....

**Reply by magicbug** [reply]
Cool answers that question.. in that case screenshots of it all :)

**Comment by phl0** [comment]
Found it. The error happens if you set arbitrary data into the locator field of you user profile:

![Screenshot from 2022-10-27 14-52-20](https://user-images.githubusercontent.com/7112907/198289632-a8c3194d-15b5-448f-a051-b6773e46d3a9.png)

Result:

![Screenshot from 2022-10-27 14-53-40](https://user-images.githubusercontent.com/7112907/198289710-3a8ca353-7848-48ab-9700-9b6e26339311.png)

So the root cause should be an invalid locator in your user profile.

**Comment by G0POA** [comment]
Awesome, I’ll take a look later and report back


On 27 Oct 2022, at 13:54, Florian (DF2ET) ***@***.***> wrote:

﻿

Found it. The error happens if you set arbitrary data into the locator field of you user profile:

[Screenshot from 2022-10-27 14-52-20]<https://user-images.githubusercontent.com/7112907/198289632-a8c3194d-15b5-448f-a051-b6773e46d3a9.png>

Result:

[Screenshot from 2022-10-27 14-53-40]<https://user-images.githubusercontent.com/7112907/198289710-3a8ca353-7848-48ab-9700-9b6e26339311.png>

So the root cause should be an invalid locator in your user profile.

**Comment by phl0** [comment]
I have created a patch against dev (https://github.com/magicbug/Cloudlog/pull/1725) to deal with this.
